portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -117,6 +117,27 @@ definition {
 			locator1 = "FormFields#COLOR_SWATCH_ITEM");
 	}
 
+	macro editConfirmation {
```

Comment:
@Jamersonze can you please rename this macro to editRequireConfirmationField? Thanks!

Reply:
Renamed